app/presenters/user_presenter.rb

```diff
@@ -0,0 +1,31 @@
+class UserPresenter < SimpleDelegator
```

Comment:
One way I like to think about decorators/presenters is that they're tied to a specific context. For example, this one is a wrapper around User for displaying of roles, but many of these methods would not be useful in other contexts. Something to continue, but this could be `UserUserRolePresenter` or possibly organize by the thing being decorated: `User::GlobalRolePresenter` or something along those lines. I don't feel strongly, but it's something to consider.